docs/extending/extending_draftail.md

```diff
@@ -303,7 +303,7 @@ const Stock = (props) => {
 
 This is a straightforward React component. It does not use JSX since we do not want to have to use a build step for our JavaScript.
 
-Finally, we register the JS components of our plugin:
+Finally, we register the JS components of our plugin *note this is ran outside of an event listener such as DOMContentLoaded*:
```

Comment:
I'm concerned this line makes it more confusing, I understand you are talking about event listener in the context of the above code (I think). 

However, you advise that the registration should happen outside of an event listener and then suggest DOMContentLoaded which itself is an event. 

Are you advising that you need to register a plugin before content loaded or after? Or just saying it should be registered outside of any Callback?

Maybe a suggestion would be to add a comment line to the JS snippet below with where to put the code?

Sorry if I'm confused about the problem here but we just want to ensure we add clarity. 

As an aside; we don't use lots of bold/italics but rather admonitions if needed or just plain sentences. However, in this case I think a comment line in the JS snippet may be a good place for any clarification needed. 

Example
````
```javascript
// do not call this within any event listener, instead call as early as possible 
window.draftail.registerPlugin({
    type: 'STOCK',
```
````

Note. Not saying this text is right - just an example of where to put the comment.

Reply:
```suggestion
Finally, we register the JS components of our plugin:
```